index.js

```diff
@@ -16,7 +16,7 @@ var defaultOptions = {
     }
 };
 
-module.exports = fs => {
+const prep = fs => {
```

Comment:
Can you change this to `function prep(fs)`?
I want to keep the next minor release backwards compatible.